packages/main/pyproject.toml

```diff
@@ -39,7 +39,7 @@ rpaframework-core = "^11.0.1"
 rpaframework-pdf = "^7.1.5"
 rpaframework-windows = { version = "^7.3.2", platform = "win32" }
 jsonpath-ng = "^1.5.2"
-robotframework = ">=4.0.0,!=4.0.1,<6.0.0"
+robotframework = ">=4.0.0,!=4.0.1,<7.0.0"
```

Comment:
Hey... just jumping in after the meeting. 😊 
If we'd like to avoid `6.1.0` should these clauses be more restrictive?

Reply:
Yeah I will add 6.1.0 exclusion.